src/App.js

```diff
@@ -2,32 +2,67 @@ import React, { useState, useEffect } from "react";
 
 import "./App.css";
 
+class post{
```

Comment:
We don't really need a whole new class for a post. It can be just a dictionary/map/object
`const post = { message: "hello", name: "DANIEL", color: "#0fa000" }`